website/content/en/docs/migration/v0.19.3.md

```diff
@@ -0,0 +1,13 @@
+---
+title: v0.19.3
+weight: 999980997
+---
+
+## Upgrade your project from version "2" to "3-alpha"
+
+The SDK's default Go plugin no longer supports OLM-related project files nor writes a `plugins` PROJECT field for projects scaffolded previously with `operator-sdk init --project-version=2`, Please migrate to project version "3-alpha" for support of these features by adding the following to your `PROJECT` file:
+```yaml version: "3-alpha" # Updated from "2" layout: go.kubebuilder.io/v2 plugins:
+  go.sdk.operatorframework.io/v2-alpha: {}
+```
```

Comment:
This doesn't seem to render properly. It should look like this:

`````suggestion
```yaml
version: "3-alpha" # Updated from "2"
layout: go.kubebuilder.io/v2
plugins:
  go.sdk.operatorframework.io/v2-alpha: {}
```
`````